dask_elk/elk_entities/node.py

```diff
@@ -33,20 +33,29 @@ def __init__(self):
     def nodes(self):
         return self.__nodes
 
-    def get_nodes_from_elastic(self, elk_client):
+    def get_nodes_from_elastic(self, elk_client, wan_only):
         """
+        if wan_only value is True, then set None on publish_address
```

Comment:
Please add it as parameter in the doc string bellow like `elk_client` e.g

```python
:param bool wan_only: Whether DaskElkClient is connected in wan_only mode or not
```
This doc strings is necessary for the documentation to be correctly created. 
You can add it under the `elk_client` param entry in the doc string